.gitignore

```diff
@@ -64,3 +66,5 @@ _region_.tex
 
 # master_data archive folders
 .archive
+
+local.css
```

Comment:
Yep, this is for placing local-only styling. With axiom now being packaged, it can be hard to tell if you are running a version that was installed from a package or a linked one. In that case, I think it's nice to  have some local only styling, like a placeholder on the bottom or the top, that gives you a visual cue that you are working on a linked version.

Reply:
ok, cool. I'll keep it in my notes to remember to use when I'm developing